src/cmd-build-validate

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import json
+import argparse
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+from cosalib.builds import Builds
+from cosalib.cmdlib import sha256sum_file
+
+NONDISK_FORMATS = ['ostree', 'iso']
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID")
+parser.add_argument("--image", help="Just verify given image type")
+parser.add_argument("--no-checksum", help="Skip validating checksum", action='store_true')
+args = parser.parse_args()
+
+
+def validate_build(builddir):
+    buildmeta_path = os.path.join(builddir, 'meta.json')
+    with open(buildmeta_path) as f:
+        buildmeta = json.load(f)
+
+    found = args.image is None
+    for img_format, data in buildmeta['images'].items():
+        if args.image is not None and img_format != args.image:
+            continue
+        found = True
+        if img_format in NONDISK_FORMATS:
+            print(f"Skipping non-disk artifact: {img_format}")
```

Comment:
We could still validate the checksum though, right?

Reply:
For the OSTree one, would be cool to unpack and `ostree fsck` too, though don't need to do that here.